tests/runtime/device_runtime_tests.cpp

```diff
@@ -0,0 +1,312 @@
+#include <chrono>
+// #include <filesystem>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <stdarg.h>
+#include "protocol_nmea.h"
+#include "device_runtime_tests.h"
+#include "ISFileManager.h"
+
+
+#define LOG_DIRECTORY   "realtime_logs"
+
+DeviceRuntimeTests::DeviceRuntimeTests()
```

Comment:
While I generally don't feel as strongly about Doxygen stuff in our tests as I do in the mainline code, I do think its a good practice all the way around... So, I won't reject it, but I'd like to encourage all of us to add Doxygen for all code (since eventually, someone else will have to access/modify it, etc).